set.mm

```diff
@@ -268465,7 +268503,11 @@ Additional definitions for (multivariate) polynomials
     mhp0cl.i $e |- ( ph -> I e. V ) $.
     mhp0cl.r $e |- ( ph -> R e. Grp ) $.
     mhp0cl.n $e |- ( ph -> N e. NN0 ) $.
-    $( The zero polynomial is homogeneous.  (Contributed by SN,
+    $( The zero polynomial is homogeneous.  Under the current definition, it
+       has any (nonnegative integer) degree which loosely corresponds to the
+       value "undefined".  The values ` -oo ` and ` 0 ` are also used in
+       Metamath (by ~ df-mdeg and ~ df-dgr respectively) and the literature:
```

Comment:
Zero sounds like a strange choice. Could we modify df-dgr by simply replacing NN with ZZ ?

Reply:
df-sup simplifies to `U. { x e. ZZ | A. y e. ZZ -. y < x }` when `A = (/), B = ZZ, R = <` (`A` is the set of coefficients which is `(/)` for a zero polynomial)

so the degree would become `(/)` since unlike NN0, ZZ does not have a minimum

Reply:
Oh, I didn't check the set.mm definitions. Then it's not $\mathbb{Z}$ one should use, but  $\overline{\mathbb{Z}}$, of course.

Reply:
Thanks for #4102 ! Resolving.